tests/solids/LiH_solid_1x1x1_pp/CMakeLists.txt

```diff
@@ -50,7 +50,7 @@
                     0 LIH_X_SCALARS # VMC
                     )
 
-IF(ENABLE_SOA)
+IF(ENABLE_SOA OR QMC_CUDA)
```

Comment:
The logic is still strange here. If we decide to port the new hybrid to CPU, then ENABLE_SOA is always needed and it should be OR.

Reply:
Since I want to see that the  hybrid rep code is not working at all with CUDA, the logic is correct. I wrote these when I thought the hybrid rep code had not been rewritten  and the CUDA implementation broken, but that does not change the logic.